cmake/dependencies.cmake

```diff
@@ -26,22 +26,29 @@ fetch_models_and_validation_set()
 get_linux_name(LINUX_OS_NAME)
 
 if(CMAKE_CROSSCOMPILING AND CMAKE_HOST_SYSTEM_NAME MATCHES Linux AND CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "amd64.*|x86_64.*|AMD64.*")
```

Comment:
am I right that compiling for `CHROMIUMOS` is a cross-compilation? should we create a cmake toolchain for this case? Like here https://github.com/openvinotoolkit/openvino/tree/master/cmake/toolchains